### PR TITLE
chore: Merge next-minor into next-major

### PR DESCRIPTION
**Pull request by gravwell-son-of-logbot**
This PR addresses no issue.

It proposes merging `next-minor` into `next-major` via `merge/next-minor_to_next-major`.

### CODE REVIEW

File: .github/workflows/golang-testing.yml [hunk 1]
```diff
@@ -28,7 +28,7 @@ jobs:
     - name: Set up Go
       uses: actions/setup-go@v4
       with:
-        go-version: 1.21.12
+        go-version: 1.21.13
```
Comment (ashnwade) [inline .github/workflows/golang-testing.yml:31]:
```suggestion
        go-version: 1.23.0
```

File: ingesters/version/version.go [hunk 2]
```diff
@@ -16,12 +16,12 @@ import (
 
 const (
 	MajorVersion = 5
```
Comment (ashnwade) [inline ingesters/version/version.go:18]:
```suggestion
	MajorVersion = 6
```

File: ingesters/version/version.go [hunk 3]
```diff
@@ -16,12 +16,12 @@ import (
...
-	MinorVersion = 5
+	MinorVersion = 6
```
Comment (ashnwade) [inline ingesters/version/version.go:19]:
```suggestion
	MinorVersion = 0
```

File: ingesters/version/version.go [hunk 4]
```diff
@@ -16,12 +16,12 @@ import (
...
 	PointVersion = 0
 )
 
 var (
-	BuildDate time.Time = time.Date(2024, 7, 24, 0, 0, 0, 0, time.UTC)
+	BuildDate time.Time = time.Date(2024, 8, 21, 0, 0, 0, 0, time.UTC)
```
Comment (ashnwade) [inline ingesters/version/version.go:24]:
```suggestion
	BuildDate time.Time = time.Date(2024, 12, 10, 0, 0, 0, 0, time.UTC)
```

**Review by ashnwade — CHANGES_REQUESTED** [review]
Suggested changes for major version

**Review by david-fritz-gravwell — APPROVED** [review]
LGTM

**Pull request merged**